twig/js/form_tittellos.html.twig

```diff
@@ -44,8 +44,12 @@
 				<div class="form-group text-danger">
 					<label for="kommune">Kommune/bydel *</label>
 					<select name="kommune" class="form-control" required>
-						{% for id, navn in monstring.kommuner %}
-							<option value="{{ id }}" {{ innslag.kommune_id == id ? 'selected="selected"' : '' }}>{{ navn }}</option>
+						{% for fylke in fylker %}
+							<optgroup label="{{ fylke.navn }}">
+								{% for kommune in fylke.kommuner %}
+									<option value="{{ kommune.id }}" {{ innslag.kommune_id == kommune.id ? 'selected' : '' }}>{{ kommune.navn }}</option>
+								{% endfor %}
+							</optgroup>
```

Comment:
Har du testet dette på et arrangement som ikke har "land" som nedslagsfelt @slogan-15 ?
Jeg mistenker at de ikke har fylker-variabelen (det gir i alle fall mening at de ikke skal ha det), og at det nå ikke funker for de.

I tilfelle er nok løsningen å bruke
```twig
{% if nedslagsfelt == 'land' %}
  den nye koden
{% else %}
  den gamle koden
{% endif %}

Reply:
Det er ikke noe feilmelding med andre typer i hvertfall. Fylker-variablen kommer ikke fra arrangementet, det kommer fra klassen Fylker.